docs/csharp/getting-started/with-visual-studio.md

```diff
@@ -1,5 +1,6 @@
 ---
 <<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
That doc gave me trouble for some reason even though the names are totally different. Should be fixed now.